src/curated-corpus/components/actions/ScheduleCorpusItemAction/ScheduleCorpusItemAction.test.tsx

```diff
@@ -44,7 +44,7 @@ describe('The ScheduleCorpusItemAction', () => {
   });
 
   it('completes the action successfully', async () => {
-    const today = DateTime.local();
+    const today = DateTime.local().plus({ days: 1 });
```

Comment:
Looks good! I was wondering if it's worth adding a comment here for posterity why this change was made

Reply:
will do !